### DISCUSSION TITLE
How this UI inside docker related to Kafka service on server?

### DISCUSSION THREAD

**Discussion by AAlex-11** [provectus/kafka-ui#4021] [Q&A] [answered]
I'm very sorry, I'm newbie in Kafka, I have installed Kafka on my server from this instruction - https://www.digitalocean.com/community/tutorials/how-to-install-apache-kafka-on-ubuntu-20-04 and it worked perfectly. And now I'm search UI for my Kafka installation. But I see something software in Docker (isolated from my computer by Docker), and I don't understand how this UI will communicate with service in my computer.

**Comment by Haarolean** [comment, marked as answer]
https://github.com/provectus/kafka-ui/discussions/1081
check this
